### PR TITLE
docs: make readme more beautiful and readable

### PR DESCRIPTION
**Pull request by riivx**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,6 +58,8 @@ Bloxstrap also only runs whenever necessary, so it doesn't stay running in the b
 * [@MaximumADHD](https://github.com/MaximumADHD) - Initially inspiring the idea for Bloxstrap with [Roblox Studio Mod Manager](https://github.com/MaximumADHD/Roblox-Studio-Mod-Manager).
 * [Multako](https://www.roblox.com/users/2485612194/profile) - Designing the Bloxstrap logo.
 * [@1011025m](https://github.com/1011025m) - Providing a method for disabling the Roblox desktop app.
-* taskmanager ([@Mantaraix](https://github.com/Mantaraix)) - Helping with designing the new menu look and layout.
+* taskmanager (@Mantaraix](https://github.com/Mantaraix)) - Helping with designing the new menu look and layout.
```
Comment (bluepilledgreat) [inline README.md:61]: you broke it

**Review by bluepilledgreat — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (riivx) [reply README.md]: my bad, fixed

File: README.md [hunk 2]
```diff
@@ -59,3 +61,5 @@ Bloxstrap also only runs whenever necessary, so it doesn't stay running in the b
 * taskmanager ([@Mantaraix](https://github.com/Mantaraix)) - Helping with designing the new menu look and layout.
 * [@Extravi](https://github.com/Extravi) - Allowing their presets to be bundled with Bloxstrap, and helping with improving UX.
 * [@axstin](https://github.com/axstin) - Making [rbxfpsunlocker](https://github.com/axstin/rbxfpsunlocker), which was used for Bloxstrap's FPS unlocking up until v2.2.0.
+
+[^1]: This happens because it's an unknown program, not because it's actually detected as being malicious.
```
Comment (pizzaboxer) [inline README.md:65]: Should this really be a footnote? I feel like this is a pretty important detail to outline.

File: README.md [hunk 3]
```diff
@@ -8,25 +8,26 @@
 
 An open-source, feature-packed alternative bootstrapper for Roblox.
 
-This a drop-in replacement for the stock Roblox bootstrapper, working more or less how you'd expect it to, while providing additional useful features. This does not touch or modify the game client itself, it's just a launcher! So don't worry, there's practically no risk of being banned for using this.
+>[!NOTE]
```
Comment (pizzaboxer) [inline README.md:11]: I don't really like the use of a banner section here, I might remove that.

File: README.md [hunk 4]
```diff
@@ -38,9 +39,10 @@ Here's some of the features that Bloxstrap provides over the stock Roblox bootst
 * Lets you see what region your current server is located in
 * Lets you have multiple Roblox game instances open simultaneously
 
-All the available features are browsable through the Bloxstrap menu.
+All the available features are browsable through the Bloxstrap settings menu.
 
-Bloxstrap also only runs whenever necessary, so it doesn't stay running in the background when you're not playing.
+>[!TIP]
+>Bloxstrap also only runs whenever necessary, so it doesn't stay running in the background when you're not playing.
```
Comment (pizzaboxer) [inline README.md:45]: This too, this is supposed to be touted more as a feature than a 'tip', as other modding tools that came before Bloxstrap **do** stay running in the background when you're not playing (rbxfpsunlocker, etc)

Comment (riivx) [reply README.md:45]: maybe note will be better here?

File: README.md (hunk 2 above)
Comment (riivx) [reply README.md:65]: i thought it was explanation, that's why footnote
if you want so, i can definitely change that

File: README.md (hunk 3 above)
Comment (riivx) [reply README.md:11]: understood, if you want, i can change that too

**Comment by pizzaboxer** [comment]
I won't merge this since a lot of this seems fairly superfluous, and there's some rearranging of my own I'd be doing that would conflict with this anyway.

**Pull request closed without merging**